src/main.ts

```diff
@@ -12,6 +12,23 @@ export function getPageHtml(page: Page) {
   }, config.selector);
 }
 
+function shouldCrawl(url: string): boolean {
+  // This function checks if a given URL should be crawled or not.
+  // It returns false if the URL contains any of the directories specified in the exclude array of the config object.
+  // Otherwise, it returns true.
+
+  // Iterate over each directory in the exclude array of the config object
+  for (const dir of config.exclude) {
+    // If the URL contains the current directory, return false
+    if (url.includes(dir)) {
```

Comment:
for consistency sake, can we make these glob expressions? similar to include. we can use minimatch for testing against it